content/api/commands/task.md

```diff
@@ -242,11 +253,20 @@ same-origin URLs.
 We can save the variable and retrieve the saved variable outside of the test
 using `cy.task()` as shown below.
 
-```javascript
-// in test
+:::cypress-visit-mount-test-example
```

Comment:
Oh yeah. This is confusing. I think I added this one because it's capturing the concept of using a variable outside the test. But the code example doesn't hold up across both testing types. 

Is it worth updating the content of the example to be applicable for both testing types? Or just drop it?

Reply:
I don't think it can be converted to a component test, the entire concept of visiting URLs is E2E-specific.